tools/learnset_helpers/teachable.py

```diff
@@ -3,6 +3,11 @@
 import json
 import os
 
+# before all else, abort if the config is off
+with open("./include/config/pokemon.h", "r") as file:
+    if re.findall("#define P_LEARNSET_HELPER_TEACHABLE *([^ ]*)", file.read())[0] != "TRUE":
+        quit()
```

Comment:
There's no handling in case `P_LEARNSET_HELPER_TEACHABLE` doesn't exist.
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/2904965/22fa59c3-d0bb-48f0-b4f7-ab41b752a520)

Reply:
Good catch!